src/tests/Register.test.js

```diff
@@ -64,4 +65,80 @@ it('allows the user to register successfully', async () => {
 
     expect(screen.getByLabelText('response')).toHaveTextContent("User was created successfully. A confirmation email has been sent via email. After confirming your email you can login.")
     
-})
\ No newline at end of file
+})
+
+it('Check empty field warning', async () => {
```

Comment:
please use the present tense with lower case so it reads well as a test name (`it checks empty field warning` instead of `it Check empty field warrning`)